.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: Run python tests
+
+on:
+  # Make workflow callable.
+  workflow_call:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: ["3.10"]
+        test-case:
+          - name: Check Turtle syntax - shapes/plot-description/slope/shapes.ttl
+            command: ontotools file validate shapes/plot-description/slope/shapes.ttl
+          - name: Check Turtle syntax - shapes/plot-description/slope/invalid.ttl
+            command: ontotools file validate shapes/plot-description/slope/invalid.ttl
+          - name: Check Turtle syntax - shapes/plot-description/slope/valid.ttl
+            command: ontotools file validate shapes/plot-description/slope/valid.ttl
```

Comment:
I think we can use pytest to test these files too since they are defined in the manifest for the shapes test already.